src/main/java/com/kryptnostic/conductor/rpc/odata/Table.java

```diff
@@ -102,8 +102,8 @@ static CassandraTableBuilder createTableDefinition( Table table ) {
                                 CommonColumns.TIME_UUID
                         )
                         .columns(
-                                CommonColumns.DATA_ID,
                                 CommonColumns.AUDIT_ID,
+                                CommonColumns.DATA_ID,
                                 CommonColumns.BLOCK_ID
                         )
                         .sasi(
```

Comment:
Don't use sasi, use ordinary secondary index for now. (Sasi gave us pain in reading data/doing linking earlier)